libres/tests/enkf/enkf_obs_paths_detailed.cpp

```diff
@@ -0,0 +1,194 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <string>
+#include <iostream>
+
+#include <catch2/catch.hpp>
+
+#include <ert/util/test_work_area.hpp>
+#include <ert/util/test_util.h>
+
+#include <ert/enkf/enkf_obs.hpp>
+
+/*
+ * Set up workarea, create and write start of file.
+ * Returns pointer to workares since it needs to be kept alive
+ * for the actual test
+ */
+ecl::util::TestArea *setup() {
```

Comment:
Is there a way to write this test without using `ecl::util::TestArea`?
Ideally, we want to use stuff from `catch2` if we can.

Reply:
There probably is - any example of how? (Current code is adopted from existing tests.)

Reply:
I have not read up enough on this.
@dotfloat and / or @eivindjahren ?

Reply:
@dafeda  do you have any comments on @eivindjahren s rewrite? It is it along the lines of what you / Zohar have been doing?

Reply:
@eivindjahren : referring to the latest discussions on the subject of BDD-style vs traditional fixtures, should we then leave this test as it is?

Reply:
We can leave it as is now, and move it to `TmpDir` once that is merged in, which is any minute now :D